src/njoy21/input/PLOTR/src/readCurves.hpp

```diff
@@ -0,0 +1,15 @@
+template< typename Istream >
+static std::vector< std::pair< Card2, Curves > >
+ readCurves( Istream& is, Argument< PLOTR::Card1::Lori>& lori )
+{
+  std::vector< std::pair< Card2, Curves > > curveList;
+  Card2 tmp2( is, lori );
+
+  while( tmp2.iplot.value != 99 ){
+    Curves tmpc( is, tmp2.iplot.value );
+    curveList.push_back( std::make_pair(tmp2, tmpc) );
```

Comment:
I'm guessing a `std::move` statement (or two) would help performance here.

Reply:
I think the std::make_pair constructor defaults to move construction since C++14.  Could very easily be wrong though.